### ISSUE TITLE
Environment variable ZBX_MEMORYLIMIT not working

### ISSUE THREAD

**Issue by pedropilla** [zabbix/zabbix-docker#539]
I have ZBX_MEMORYLIMIT in my compose file but does nothing, I have to manually change in /etc/php7/php.ini.

 It is a very simple issue, so I think there isn't necessary more data, right?

**Comment by rockaut** [comment]
Can you post the container log? There should be a mention of the variable.
Which image are you using?

**Comment by pedropilla** [comment]
I'm using "zabbix/zabbix-web-nginx-mysql". In fact there is a mention of the variable, but something is going wrong because if you go check the cfg you will see the default 128MB.

Here are the logs: 
```
root@secondary:~/zabbixt# docker logs zabbix_infra-web_1
** Deploying Zabbix frontend (nginx) with mysql database
** Preparing the system
** Preparing Zabbix web-interface
** Using MYSQL_USER variable from ENV
** Using MYSQL_PASSWORD variable from ENV
********************
* DB_SERVER_HOST: 1.2.3.4
* DB_SERVER_PORT: 3306
* DB_SERVER_DBNAME: zabbix
* DB_SERVER_ZBX_USER: zabbix
* DB_SERVER_ZBX_PASS: zabbix
********************
** Disable default vhosts
** Adding Zabbix virtual host (HTTP)
**** Impossible to enable SSL support for Nginx. Certificates are missed.
** Preparing Zabbix frontend configuration file
** Updating '/etc/php7/conf.d/99-zabbix.ini' parameter "max_execution_time": '600'... updated
** Updating '/etc/php7/conf.d/99-zabbix.ini' parameter "memory_limit": '28672M'... updated
** Updating '/etc/php7/conf.d/99-zabbix.ini' parameter "post_max_size": '16M'... updated
** Updating '/etc/php7/conf.d/99-zabbix.ini' parameter "upload_max_filesize": '2M'... updated
** Updating '/etc/php7/conf.d/99-zabbix.ini' parameter "max_input_time": '300'... updated
** Updating '/etc/php7/conf.d/99-zabbix.ini' parameter "date.timezone": 'America/Sao_Paulo'... added
** Cleaning the system
########################################################
** Executing supervisord
2020-01-08 18:13:37,909 INFO Included extra file "/etc/supervisor/conf.d/supervisord_zabbix.conf" during parsing
2020-01-08 18:13:37,909 INFO Included extra file "/etc/supervisor/conf.d/supervisord_zabbix.conf" during parsing
2020-01-08 18:13:37,909 INFO Set uid to user 0 succeeded
2020-01-08 18:13:37,909 INFO Set uid to user 0 succeeded
2020-01-08 18:13:37,926 INFO RPC interface 'supervisor' initialized
2020-01-08 18:13:37,926 INFO RPC interface 'supervisor' initialized
2020-01-08 18:13:37,927 INFO supervisord started with pid 1
2020-01-08 18:13:37,927 INFO supervisord started with pid 1
2020-01-08 18:13:38,930 INFO spawned: 'nginx' with pid 40
2020-01-08 18:13:38,930 INFO spawned: 'nginx' with pid 40
2020-01-08 18:13:38,933 INFO spawned: 'php-fpm7' with pid 41
2020-01-08 18:13:38,933 INFO spawned: 'php-fpm7' with pid 41
```

**Comment by rockaut** [comment]
Can't test it currently. Will do tomorrow. Have you tried some lower values
like 256M?maybe there's a problem. If you want unlimited you should use -1
I think.

Pedro Pilla <notifications@github.com> schrieb am Mi., 8. Jan. 2020, 20:56:

**Comment by dotneft** [comment]
I can not confirm the issue at least with Alpine image:
`bash-5.0# cat /etc/php7/conf.d/99-zabbix.ini
max_execution_time=600
memory_limit=28672M
post_max_size=16M
upload_max_filesize=2M
max_input_time=300
; always_populate_raw_post_data=-1
max_input_vars=10000
; date.timezone=Europe/Riga
date.timezone=Europe/Riga
; session.save_path=/var/lib/php7
`

**Comment by marklahn** [comment]
I just want to add, that I thought I had encountered this problem too.
Running 6.4.13 or 7.0.0 alpine images, I don't see the line:
** Updating '/etc/php7/conf.d/99-zabbix.ini' parameter "memory_limit": '28672M'... updated

And I also see my /etc/php83/php.ini has memory_limit = 128M (default), so it looked like my ZBX_MEMORYLIMIT didn't work.
However, I tried a simple php file with phpinfo(), and saw my memory_limit indeed was 256M.

So php-fpm83 runs PHP with different settings than found in the php.ini file. 

Tricky to troubleshoot, but there wasn't a problem. ZBX_MEMORYLIMIT did indeed work.
